### PR TITLE
ci: add job

### PR DESCRIPTION
**Pull request by nickkbell**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            - '**'
+    pull_request:
+    # manually
+    workflow_dispatch:
+
+permissions:
+    contents: read
+
+# allow only one concurrent run
+concurrency: "test"
```
Comment (DarkPark) [inline .github/workflows/lint.yml:19]:
```suggestion
concurrency: "lint"
```

?

**Comment by DarkPark** [comment]
I see some eslint warnings. Would be nice to fix them or disable rules if necessary.

**Review by aleks-and — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (nickkbell) [reply .github/workflows/lint.yml]: ++

File: tests/fixtures/mtx/commands.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 export const uplinkCommands = {
     getCurrentStatusMeter: {
         hex: {
-            dump: '7e 51 ff fe 15 75 07 7d 33 b7 34 87 63 dc ee b2 54 4b 59 89 6a c9 f6 db 6a b1 40 46 75 d0 3a 79 74 58 4d 22 27 4d 19 79 0b 44 02 cf 0d 24 56 57 64 c8 73 5b 9e 5b 76 c0 c4 9e ce 7e',
+            dump: '7e 51 ff fe 15 75 07 7d 33 b7 34 87 63 dc ee b2 54 4b 59 89 6a c9 f6 db 6a b1 40 46 75 d0 3a 79 74 '
```
Comment (DarkPark) [inline tests/fixtures/mtx/commands.js:4]: you can use template strings here without concatenation

**Review by aleks-and — APPROVED** [review]

File: tests/fixtures/mtx/commands.js (hunk 2 above)
Comment (nickkbell) [reply tests/fixtures/mtx/commands.js:4]: template strings preserve all whitespace and breaks, one dump is parsed as multiple dumps

File: tests/specs/mtx/downlinkCommands.spec.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ test.describe('mtx downlink commands - parse hex dumps', () => {
             const mainPage = await new MainPage(page);
 
             await mainPage.parseDump(command.hex.dump);
+            // eslint-disable-next-line playwright/no-wait-for-timeout
```
[the remark this answers is not in the document]
Comment (nickkbell) [reply tests/specs/mtx/downlinkCommands.spec.js:21]: Timeouts are not recommended. if I can't find another way then I will move it

**Pull request merged**